test/RecipeRepository-test.js

```diff
@@ -1,13 +1,18 @@
 import { expect } from 'chai';
 import RecipeRepository from '../src/classes/RecipeRepository';
+import Recipe from '../src/classes/Recipe';
 import ingredientsData from '../src/data/ingredients';
 import sampleData from '../src/data/sampleData';
 
 describe('RecipeRepository', () => {
   let repository;
+  let recipeList;
 
   beforeEach(() => {
-    repository = new RecipeRepository(sampleData);
+    recipeList = sampleData.map(recipe => {
```

Comment:
Instead of importing all the data and using the 'raw' data, we realized we needed to use the Recipe class we'd created earlier. Consequently, when we bring in the sampleData, we're now creating an array of the class instance rather than bringing in that array with data.